### DISCUSSION TITLE
winget turn off auto update

### DISCUSSION THREAD

**Discussion by whitephnx1** [Romanitho/Winget-AutoUpdate#744] [General]
So i discovered that even if you have winget setting disabled to prevent it from auto updating itself, that winget now finds itself in the repo so it updates itself using that. I have had to add it into the blacklist to avoid it from updating itself via the repo. Is there another way to prevent this?

**Comment by Romanitho** [comment]
Hello,
Unfortunately, not. It is like any other app managed by winget. You have to disable auto-update from the app itself and blacklist your app in WAU.

**Reply by whitephnx1** [reply]
ok no worries. just wanted to make sure i wasn't missing something. Thank you!

**Reply by Romanitho** [reply]
Perhaps we can add WAU to the default blacklist since WAU has its own self-update system.

**Reply by whitephnx1** [reply]
Totally up to you. For groups who have to verify each update before it is pushed, we have to make sure the app can't update itself while we review. Have a great day!

**Reply by whitephnx1** [reply]
FYI I do love the work you put into this, Great job!

**Reply by Romanitho** [reply]
Thanks :)
